app/views/people/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1> People </h1>
+
+<h3> ID: Person, Total Tasks, (number incomplete)</h3>
+
+<% @people.each do |person| %>
+
+<ul>
+  <li>
+    <%= person.id %>:
+    <%= link_to "#{person.name}", "/people/#{person.id}/show"%>,
+    Tasks: <%= @tasks.where(person_id: person.id).count %>,
```

Comment:
I see that the logic on this line is duplicated on line #13 - This would be a good place to use a variable to reduce duplication